src/pages/people.js

```diff
@@ -1,8 +1,24 @@
-import { html, LitElement } from "lit";
+import { html, css, LitElement } from "lit";
 import { customElement } from "lit/decorators.js";
+import "/components/PersonItem.js";
 import { fetchPeople } from "/api.js";
 
 export class PagePeople extends LitElement {
+  static get styles() {
+    return css`
+      div {
```

Comment:
Se cambiaría al `ul` por la sugerencia de abajo.

```suggestion
      ul {
```